fr.inria.gitlabpages.coccinelle.Coccinelle.yaml

```diff
@@ -0,0 +1,30 @@
+app-id: fr.inria.gitlabpages.coccinelle.Coccinelle
```

Comment:
i can't reach coccinelle.gitlabpages.inria.fr it redirects to gitlab sign in. Do you have a webpage up there and set to public?

Reply:
ok https://coccinelle.gitlabpages.inria.fr/website/ works but not https://coccinelle.gitlabpages.inria.fr/

Reply:
https://coccinelle.gitlabpages.inria.fr/website/ is the homepage and should work without authentication.

Reply:
as long as you can put the file under https://coccinelle.gitlabpages.inria.fr/.well-known/ to verify it later, it's fine.

Reply:
Would https://coccinelle.gitlabpages.inria.fr/website/.well-known work?

Reply:
No

Reply:
https://docs.flathub.org/docs/for-app-authors/requirements#control-over-domain-or-repository

Reply:
Then we should switch to a repository-style ID.

https://github.com/coccinelle/coccinelle <-> io.github.coccinelle.coccinelle

I'll make the changes here and then upstream if things look good.

Reply:
yes that's fine.